apps/prairielearn/src/lib/course.ts

```diff
@@ -192,3 +193,31 @@ export async function pullAndUpdateCourse({
 
   return { jobSequenceId: serverJob.jobSequenceId, jobPromise };
 }
+
+/**
+ * Return the courses that the user has administrator access to.
```

Comment:
"administrator" means a specific thing to us, and it isn't really what we're looking for here. I think "staff access" is more appropriate.